themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Projects in Self-Managed State Backends"
+date: 2023-04-03
+meta_desc: Pulumi's Self-Managed backend now supports stack names scoped by project name.
+meta_image: meta.png
+authors:
+    - justin-vanpatten
+    - abhinav-gupta
+tags:
+    - features
+    - pulumi-releases
+---
+
+At Pulumi, we aim to offer our users the best-in-class IaC experience.
+We recently shipped a series of impactful enhancements to our [Pulumi Service](/product/pulumi-service/) backend
+(check out [Pulumi Deployments](/blog/pulumi-deployments/) if you haven't already!),
+and we have more exciting features coming soon.
+
+While our Service backend remains the most ideal place to store your Pulumi state,
```

Comment:
```suggestion
While we recommend using the Pulumi Service backend to store your state because of its [many features](/blog/six-things-about-pulumi-service/) that increase your productivity,
```